tn-cli/tn-cli.py

```diff
@@ -10,6 +10,7 @@
 import grpc
 import json
 from PIL import Image
+import imp
```

Comment:
I don't think it's currently being used. When I suggested `imp` I meant something like a command line parameter for optionally loading a macros file.
```
./tn-cli.py --load-macro=./path/to/macro.py
```

Reply:
Ah, makes sense. Thanks. Added the flag.